### ISSUE TITLE
Import quickfix should follow Preferred quote character setting

### ISSUE THREAD

**Issue by park9140** [TypeStrong/atom-typescript#398]

**Comment by littlebeeper** [comment]
A quickfix for this issue;
- Open Config Folder
- Find **atom-typescript/lib/main/lang/fixmyts/quickfixes/addImportFromStatements.ts**
- Change double quotes on line 47 in canProvideFix method and line 63 in provideFix method.

Hope this helps

**Comment by buehler** [comment]
Pretty sad there is an option for the prefered quote character, but the beloved autoimporter does not use it 😦 

The auto-import feature is the reason why I use atom over vscode over intelliJ 😁 

It's preeeetty awesome what you guys did!

**Comment by lordmat0** [comment]
I just recently switched back to the latest version IntellIJ as I found that atom-type auto import would sometimes work compared to IntelliJ where it would always give sensible suggestions.

Definitely worth giving it another go! (think you need to use ultimate)

**Comment by buehler** [comment]
@lordmat0 yea, I do use ultimate. But the problem with intelliJ is, that it's horrible slow like a snail on prozac.. 😠 
VSCode / Atom have a waaaay smaller memory footprint and overall resource usage than intelliJ

**Comment by Toub** [comment]
What about using prefered quote from tslint.json if it exists?

```json
{
  "rules": {
    "quotemark": [
      true,
      "single"
    ]
}}
```

**Comment by Toub** [comment]
related to #1096 #1147

**Comment by ziacik** [comment]
It seems this feature is now handled by Microsoft TypeScript directly and is fixed in ~2.6 typescript (in a little bit less than optimal way - by looking at first existing import in the file and using that style for autoimports)
See https://github.com/Microsoft/TypeScript/blob/master/src/services/codefixes/importFixes.ts#L290
